backend/app/routes.py

```diff
@@ -209,10 +209,10 @@ def aggregate_travel_times(start_node, end_node, start_time, end_time, start_dat
             JOIN congestion.network_segments_daily AS cn USING (segment_id)
             WHERE   
                 cn.hr <@ %(time_range)s::numrange
-                AND date_part('isodow', cn.dt)::integer <@ '[1, 5]'::int4range
+                AND date_part('isodow', cn.dt)::integer IN %(dow_list)s

```

Comment:
So this comes in as a string, right? But I don't see it getting parsed into a list of integers before being inserted into the query here. Did you test this? Did it seem to work?

Reply:
This comes in as a tuple of integers. It's much easier (from what I've read online) to take a python list, flatten it into a string, then use the tuple function to pass to sql, since a python list is an incompatible param type. This works for now, since the range of dow is only from 1-7. This solution works in testing. Let me know if it should be changed